hedera-node/src/test/java/com/hedera/services/store/TypedTokenStoreTest.java

```diff
@@ -109,8 +106,9 @@ void setUp() {
 				() -> tokenRels,
 				backingTokenRels,
 				uniqTokenViewsManager,
-				tokenStore::addKnownTreasury,
-				legacyStore::removeKnownTreasuryForToken);
+				legacyStore::addKnownTreasury,
+				legacyStore::removeKnownTreasuryForToken,
+				legacyStore::isKnownTreasury);
```

Comment:
instead of using 3 different "functional interfaces", lets define 1 interface with 3 methods?

Reply:
I don't think this is possible, as functional interfaces are interfaces with **single abstract method** , which can be implemented by a lambda function, as here.